.github/workflows/gh-pages.yml

```diff
@@ -72,6 +72,8 @@ jobs:
       - name: Main branch preparations
         if: github.ref == format('refs/heads/{0}', env.MAIN_BRANCH)
         run: |
+          rm -rf $GITHUB_WORKSPACE/gh-pages/js
+          rm -rf $GITHUB_WORKSPACE/gh-pages/css
```

Comment:
I think you could actually collapse this down to one line: `rm -rf $GITHUB_WORKSPACE/gh-pages/{css,js}

Reply:
That's a good trick. Testing again on the forked repo.